front/app/components/CampaignConsentForm/index.tsx

```diff
@@ -146,15 +146,15 @@ const CampaignConsentForm = ({
         })
       );
 
+    const consentChangesObject: Record<string, any> = Object.fromEntries(
+      Object.values(consentChanges).map((consent: IConsentChanges) => [
+        consent.campaignConsentId,
+        consent.consented,
+      ])
+    );
+
     trackEventByName(trackEventName, {
-      extra: {
-        consentChanges: Object.fromEntries(
-          Object.values(consentChanges).map((consent: IConsentChanges) => [
-            consent.campaignConsentId,
-            consent.consented,
-          ])
-        ),
-      },
+      consentChanges: consentChangesObject.toString(),
```

Comment:
This will result in `'[object Object]'`. Maybe we could JSON.stringify (not sure if 100% safe) or convert in some other way where we preserve the content?